Libraries/Components/WebView/WebView.ios.js

```diff
@@ -462,6 +462,16 @@ var WebView = React.createClass({
   },
 
   /**
+   * Evaluate JavaScript on the current page.
+   */
+  evaluateJavaScript: async function(script: string): Promise {
```

Comment:
I think Flow will soon require type parameters (not sure what they are called), so may be we have to change it to `Promise<any>`